PWGCF/FemtoUniverse/TableProducer/femtoUniverseProducerTask.cxx

```diff
@@ -440,6 +453,13 @@ struct femtoUniverseProducerTask {
                        -999., -999.,
                        -999., -999., -999.,
                        -999.); // QA for phi
+    } else if constexpr (isD0) {
```

Comment:
Maybe the better way to go would be to name the bool isPhi -> isPhiOrD0? It would minimize the amount of code ;)

Reply:
If you change that, I'm happy to approve the changes

Reply:
Done. I've changed it.